package.json

```diff
@@ -92,21 +92,27 @@
     "coverageDirectory": "./coverage/",
     "collectCoverage": true,
     "collectCoverageFrom": [
-      "src/**/*.js"
+      "src/**/*.js",
+      "src/**/*.ts"
     ],
     "testMatch": [
-      "**/*.(spec).(js)"
+      "**/*.(spec).(js|ts)"
     ],
     "setupFiles": [
       "./test/setupTests.js"
     ],
+    "moduleFileExtensions": [
+      "js",
+      "ts"
+    ],
     "transformIgnorePatterns": [],
     "moduleNameMapper": {
       "^#ansi-styles$": "<rootDir>/node_modules/chalk/source/vendor/ansi-styles/index.js",
       "^#supports-color$": "<rootDir>/node_modules/chalk/source/vendor/supports-color/index.js",
       "@utils/(.*)$": "<rootDir>/src/utils/$1",
       "@commands/(.*)$": "<rootDir>/src/commands/$1",
-      "@constants/(.*)$": "<rootDir>/src/constants/$1"
+      "@constants/(.*)$": "<rootDir>/src/constants/$1",
+      "^(\\.\\.?\\/.+)\\.js$": "$1"
```

Comment:
This is to allow importing files without specifying the `.js` extension right?

Reply:
This is required to make sure `jest` can resolve the new `.js` suffix imports. If we don't specify it the tests will fail because it can't find the modules.

Reply:
I see, so part of the transition to make this an ESM (since it's a requirement to specify extension on ESMs)